core/src/main/res/values/strings.xml

```diff
@@ -24,4 +24,5 @@
     <string name="device_view_title">Device Information</string>
     <string name="logs_view_title">Logs</string>
     <string name="file_issue_view_title">File Issue</string>
+    <string name="placeholder_device_info">Temporary text to be replaced by device info</string>
```

Comment:
Hm. Maybe something like "Loading device info..." ?

Reply:
That String resource will be removed when I write the code to show the actual device information.